### PR TITLE
Help modal setup for translations

### PR DESCRIPTION
**Pull request by mikeallbutt**

### CODE REVIEW

**Comment by corsacca** [comment]
One issue:
image

File: dt-contacts/contacts-post-type.php [hunk 1]
```diff
@@ -841,6 +841,7 @@ public function get_channels_list() {
                 "icon" => get_template_directory_uri() . "/dt-assets/images/skype.svg",
                 "hide_domain" => true
                 ],
+                "icon" => get_template_directory_uri() . "/dt-assets/images/skype.svg",
```
Comment (corsacca) [inline dt-contacts/contacts-post-type.php]: @mikeallbutt this icon is now here twice?

**Comment by corsacca** [comment]
@mikeallbutt did you fix the subassigned modal?

**Comment by mikeallbutt** [comment]
> @mikeallbutt did you fix the subassigned modal?

Yes @corsacca

**Comment by corsacca** [comment]
@mikeallbutt i still get the error.

**Pull request merged**